### PR TITLE
i18n(fr): Add `guides/cms/flotiq.mdx` from #9798

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/cms/flotiq.mdx` from #9798

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/flotiq.mdx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+[Flotiq](https://flotiq.com) est un CMS headless conçu pour différents frontends, tels que les sites statiques, les applications mobiles et les applications web. Les développeurs et les créateurs de contenu gèrent et diffusent le contenu par le biais d'API REST et GraphQL.
+
+## Intégration avec Astro
+
+Ce guide permet d'utiliser l'API Flotiq headless CMS avec un projet Astro pour afficher du contenu sur votre site web.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/flotiq.mdx:16]: The wording seems a bit strange to me, perhaps we can find something simpler to read:
```suggestion
Ce guide permet d'utiliser l'API CMS headless de Flotiq avec un projet Astro pour afficher du contenu sur votre site web.
```

File: src/content/docs/fr/guides/cms/flotiq.mdx [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+### Pré-requis
+
+Pour commencer, vous aurez besoin de
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/flotiq.mdx:20]: nit:
```suggestion
Pour commencer, vous aurez besoin de :
```

File: src/content/docs/fr/guides/cms/flotiq.mdx [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+
+Connectez-vous à votre compte Flotiq et créez une définition de type de contenu personnalisée avec l'exemple de configuration `Blog Post` suivant :
+- **Label** : Blog Post
+- **API Name** : blogpost
+- **Fields** :
+  - **Title** : texte, obligatoire
+  - **Slug** : texte, obligatoire
+  - **Content** : texte enrichi, obligatoire
+
+Ensuite, créez un ou plusieurs exemples d'[objets de contenu](https://flotiq.com/docs/panel/content-objects/) en utilisant le type « Blog Post ».
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/flotiq.mdx:46]: nit, the original version uses backticks and in the previous occurrence you left the backticks, so:
```suggestion
Ensuite, créez un ou plusieurs exemples d'[objets de contenu](https://flotiq.com/docs/panel/content-objects/) en utilisant le type `Blog Post`.
```

**Review by ArmandPhilippot — COMMENTED** [review]
LGTM, I just left a few nitpicks...

File: src/content/docs/fr/guides/cms/flotiq.mdx [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+
+```ini title=".env"
+FLOTIQ_API_KEY=__YOUR_FLOTIQ_API_KEY__
+```
+
+### Définir un type de contenu dans Flotiq
+
+Tout d'abord, vous devez définir un exemple de [Content Type Definition](https://flotiq.com/docs/panel/content-types/) dans Flotiq pour stocker des données.
+
+Connectez-vous à votre compte Flotiq et créez une définition de type de contenu personnalisée avec l'exemple de configuration `Blog Post` suivant :
```
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/guides/cms/flotiq.mdx:38]: Aaah yeah, miss this one.. 😆

**Comment by thomasbnt** [comment]
Thanks @ArmandPhilippot for your review!

**Review by ArmandPhilippot — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**